prometheus/base/prometheus.rules.yaml

```diff
@@ -7,11 +7,12 @@ metadata:
   namespace: o11y
```

Comment:
this has to be on openshift-user-workload-monitoring namespace. or any namespace that we want to apply this rule file to?

Reply:
It applies to all namespaces, it should be applied for this namespace as it is the namespace of the user workload Prometheus instance

Reply:
Thanks Alex